### PR TITLE
hi nikhil i have done some change

### PR DESCRIPTION
**Pull request by Cksuraj**
Hi nik i hope now we get how to do this all things

### CODE REVIEW

File: Index.html [hunk 1]
```diff
@@ -12,6 +12,9 @@
     <div class="container">
         <div class="bx"></div>
         <div class="bx"></div>
+        <div class="barbad"></div>
```
Comment (00123nikhil) [inline Index.html:15]: do not add here barbadi we will barbad already.

**Review by 00123nikhil — CHANGES_REQUESTED** [review]
please change the barbad line to make us barbad. (mi barbad zalo )

**Comment by Cksuraj** [comment]
i have made changes.

**Review by 00123nikhil — COMMENTED** [review]
great Work suraj

**Review by 00123nikhil — APPROVED** [review]
i  merged in main branch as per your request

**Pull request merged**